Bottom: bda1ecd0bc3d2f999043364042c02e44bfad43db
Top:    344b2f28b94fbd735b3962a4688b6be0cdee3e5e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:20 +0000

Bump circle approximation detail level for gerber export


---

diff --git a/src/polygon.h b/src/polygon.h
index 58634f7..90d0d26 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 24 //20 //8 //40
+#define POLY_CIRC_SEGS 40 //24 //20 //8 //40
 #define POLY_CIRC_SEGS_D ((double)POLY_CIRC_SEGS)
 
 #if 0
